examples/complete/main.tf

```diff
@@ -106,6 +106,89 @@ module "efs" {
   tags = local.tags
 }
 
+module "efs_archive" {
```

Comment:
There is no need to have this new module call in the example. If you want, update the main example.